src/models/line.js

```diff
@@ -145,23 +145,29 @@ nv.models.line = function() {
                 .data(function(d) { return [d.values] });
 
             linePaths.enter().append('path')
+              .each(function (d, i) {
+                d3.select(this)
                 .attr('class', 'nv-line')
                 .attr('d',
                     d3.svg.line()
-                    .interpolate(interpolate)
+                    .interpolate(interpolate[d[0].series])
```

Comment:
I'm confused, where is the "interpolate" array coming from?  It's still defined above as just "linear" ?